### ISSUE TITLE
Does not support Chinese characters?

### ISSUE THREAD

**Issue by wangbiye** [tulios/json-viewer#113]

**Comment by tulios** [comment]
Hi @wangbiye, the extension works with chinese. Can you check this file: https://raw.githubusercontent.com/tulios/json-viewer/master/tests/test.json

![screen shot 2016-11-14 at 09 38 37](https://cloud.githubusercontent.com/assets/33231/20258090/26e82158-aa4e-11e6-8611-4043f9be52af.png)

it might be something else, can you give more details of what is happening?

**Comment by jimmyko** [comment]
I can confirm that the Chinese can be displayed correctly. I would suggest to add Traditional Chinese in the test as well.

```json
{
  "simplified_chinese_string": "没有更多的问题",
  "traditional_chinese_string": "沒有更多的問題"
}
```

**Comment by tulios** [comment]
Thanks @jimmyko! I'll definitely add them to the test file.

**Comment by wangbiye** [comment]
Perhaps the text coding problem.
There is a new problem, it does not work. Is it because the content is too much?
http://serviceyatai.bonapp.cn/search/restaurant/do_search?userIdStr=%2523EDN3ITNzQTO%2523%2523QM5EjN1MjNzQTO3QTM&page=1&longitude=121.457680&latitude=31.212355&lang=1&globalAgentType=1&appVersion=4.8.6&appVersionCode=206&cityId=21&devUuid=520574AF-974D-42CB-8968-A35A5C3760B6&sort=rating.desc&distance=500

**Comment by tulios** [comment]
@wangbiye this URL is returning `Content-Type:text/html;charset=UTF-8`, check issue #108 for more information

**Comment by tulios** [comment]
PR #117 should improve this behavior, I'll release a new version with the changes soon

**Comment by tulios** [comment]
The improvements were released on version 0.16.0
